### ISSUE TITLE
Profil sayfası

### ISSUE THREAD

**Issue by berkerpeksag** [pyistanbul/website#5]
Haskellers ya da DjangoPeople formatında ama başlangıç için çok daha az özellikleri olan, insanların özgeçmişlerine, Twitter ya da bloglarına koyabilecekleri bir sayfa tasarlamak güzel olur.
- Adres formatı: pyistanbul.org/people/`<kullanıcı_adi>`
- Sunum yapanlar varsa onları da listelemek için Sunum modeliyle bağlayabiliriz
- GitHub Badge koyabiliriz (reklamları izlediniz)

**Comment by bahattincinic** [comment]
isi uzerine almayan varsa ben alabilirim.

**Comment by berkerpeksag** [comment]
@bahattincinic süper :) Bunu bir de insanlar sayfasına bağlamak gerekecek. Üye olduktan sonra insanlar sayfasında "aha ben buyum" deyip merge edilebilen bir şey düşünmek lazım.

Bir de, hangi bilgileri koyacağımıza karar vermek lazım. Blog, twitter ve GitHub insanlar sayfasında zaten var. Bir de StackOverflow gibi kafalarına göre doldurabilecekleri bir textarea ekleyebiliriz.

**Comment by bahattincinic** [comment]
Merge etmek konusunda şöyle bir şeyde yapabiliriz.
Suan ekli olan kullanıcılara mail gönderelim girip şifrenizi tanımlayın diye. Sanki şifremi unuttum işlemi yapıyor gibi.

**Comment by berkerpeksag** [comment]
Ben pek izinsiz e-posta gönderme taraftarı değilim. Üye olurken verilen e-posta adresi People modelindeki email alanı ile eşleşiyorsa "merge edek mi qrdş" diyebiliriz belki?

**Comment by bahattincinic** [comment]
Aynen bu daha mantıklı oldu.
Peki kisiler bölümünü Listelerken **User** ve **People** modelindeki verileri mi merge edip listelicez ?

**Comment by berkerpeksag** [comment]
Evet, muhtemelen bir süre öyle yaparız. Tabii bu durumda People modelini de read-only yapmak gerekecek. "Yeni kişi ekle" linki yerine üye kayıt sayfasını gösteririz.

**Comment by bahattincinic** [comment]
Birde ekstra olarak benim yaptigim UI cok guzel olmayabilir :smile: @fatiherikli bir ustunden gecerse guzel olur.

**Comment by berkerpeksag** [comment]
UI işini hep beraber hallederiz ya. @cihann gibi bir UI ve UX wizard'ımız var :trollface:

**Comment by cihann** [comment]
"login with github" haricinde bir social login olayi koymayalim degil mi? bence sadece github yeterli. github'i olmayan gelmesin zaten :P

**Comment by berkerpeksag** [comment]
+1

**Comment by berkerpeksag** [comment]
gerçi twitter'ı 4-5 satırda ekleyebiliyorsak ekleyebiliriz bence

**Comment by bahattincinic** [comment]
**O zaman yapilacaklari su listeyle toparliyorum:**
- Uye Giris
- Uye Kayit
- Sifremi Unuttum
- People ile User merge edilmesi
- Kullanici ile sunumlarin baglanmasi
- Social Auth

Var mi eksik kalan birsey ?

**Comment by berkerpeksag** [comment]
Sanırım yok. Bundan sonrasını da zaten ayrı issue açıp hallederiz eğer olursa. İlk etapta bu kadarı yeterli. Hatta iki parçaya bile bölebiliriz belki.

**Comment by cihann** [comment]
PM @berkerpeksag 'dan "fazlara bolelim abi" aciklamasi da geldi. :trollface:

**Comment by berkerpeksag** [comment]
mshauhsz kardeş biz developer dostu adamız, iş yükü fazla olmasın diye bölebiliriz dedim......

**Comment by bahattincinic** [comment]
Ekstra bir soru daha soriyim o zaman. Sunumlar adminden eklenip mi kullaniciya baglanicak. yoksa kullanici kendisi ekleyebilsin mi ?

**Comment by berkerpeksag** [comment]
Admin'den biz ekleriz.

**Comment by berkerpeksag** [comment]
ne durumdayız™ bahattin :P

**Comment by bahattincinic** [comment]
Unuttum ben bunu :smile:  bu hafta yollucam :smile:

**Comment by bahattincinic** [comment]
Yukaridaki maddelerde User ile People merge edilmesi disindakileri yaptim.

@berkerpeksag  Yukariki maddelerden People Merge isi kaldi sadece. People modelindeki email unique degilmis. eslestirmede email 2 tane varsa ne yapalim? (Yada boyle bir kayit var mi ?)

**Comment by berkerpeksag** [comment]
Aynı e-maile sahip iki kayıt yok bildiğim kadarıyla ama unique yapabiliriz bence o alanı.

**Comment by berkerpeksag** [comment]
Bir de, kalanının acelesi yok bence sen sunumuna odaklan :)
